packages/api-reference/src/components/ApiReferenceLayout.vue

```diff
@@ -51,6 +50,7 @@ import type {
   ReferenceLayoutSlot,
   ReferenceSlotProps,
 } from '@/types'
+import { SidebarBlock, useSidebar } from '@/v2/blocks/scalar-navigation-block'
```

Comment:
can we import the `SidebarBlock` from `scalar-sidebar-block` (not `scalar-navigation-block`)?